hash/nes.xml

```diff
@@ -53667,6 +53667,24 @@ preliminary proto for the PAL version, still running on NTSC systems) or the gfx
 		</part>
 	</software>
 
+<!-- Extracted from the Caltron 9-in-1 prototype. A real Super Gun cartridge has been dumped but is unavailable. This extracted rom should be replaced with a real dump if it differs. -->
+	<software name="supergun">
+		<description>Super Gun (Asia)</description>
+		<year>1992</year>
+		<publisher>Caltron</publisher>
+		<info name="serial" value="CN-12"/>
+		<part name="cart" interface="nes_cart">
+			<feature name="slot" value="ntdec_n715021" />
+			<feature name="mirroring" value="horizontal" />
+			<dataarea name="prg" size="65536">
+				<rom name="super gun (caltron 9-in-1 extract).prg" size="65536" crc="60497e10" sha1="c572180aae637288ffde2612c2b9f27639913db8" status="baddump" />
+			</dataarea>
+			<dataarea name="chr" size="32768">
+				<rom name="super gun (caltron 9-in-1 extract).chr" size="32768" crc="551f4e52" sha1="d2234b694225ae0bb206d2139ff6dab14e6831ce" status="baddump" />
+			</dataarea>
+		</part>
+	</software>
+
```

Comment:
Please don’t add things like this.  We can add the 9-in-1 prototype, but don’t go creating entries from ROMs extracted from muti-games.

Reply:
I gathered that MAME doesn't include extracts that never existed on physical hardware. I thought this one might be okay since the PCB of the real cart (N715021) has been studied and documented. The known dump is hoarded, so it's just a matter of waiting for another cart to turn up.

Is it satisfactory to leave the entry and comment it out (so the next person doesn't have to look up silly info like the slot and serial)?

Reply:
I guess you could add it as a NODUMP, for the purpose of documenting the PCB.  Obviously it wouldn’t be working in that state.

Reply:
Oh thank you. I forgot about nodump. Updated.